web/forms.py

```
@@ -1787,3 +1792,101 @@ class StudyGroupForm(forms.ModelForm):
+                if not isinstance(features, list):
+                    raise forms.ValidationError("Features must be a list of items")
+            except json.JSONDecodeError:
+                raise forms.ValidationError("Features must be in valid JSON format") from None
+        return features
+
+    def clean_price_monthly(self) -> float:
+        price = self.cleaned_data.get("price_monthly")
+        if price == 0 and not self.cleaned_data.get("confirm_free_plan"):
+            raise forms.ValidationError(
+                "Please confirm that you intend to create a free plan by checking the confirmation box below."
+            )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)